jsonschema/cli.py

```diff
@@ -69,19 +70,49 @@ def validation_success(self, **kwargs):
 
 @attr.s
 class _PrettyFormatter(object):
+    _MESSAGE_BAR_CHAR = '═'
+    _MESSAGE_CORNER_CHARS = ('╒', '╕')
```

Comment:
Can ditch the parens here.

Reply:
OK.  It passed through the style check.  Maybe it needs a config change?  I'll update this.